baremetal_benchmarking/workloads/tensorflow_workload.py

```diff
@@ -45,29 +45,43 @@ def download_bert_models(self):
             print("\n-- Downloading BERT checkpoints models..")
             utils.exec_shell_cmd(TF_BERT_CHECKPOINTS_WGET_CMD, None)
             utils.exec_shell_cmd(TF_BERT_CHECKPOINTS_UNZIP_CMD, None)
-            utils.exec_shell_cmd(TF_BERT_FP32_MODEL_WGET_CMD, None)
 
-        print("\n-- Required BERT models downloaded / already present..")
+        # We are deleting the model to avoid complication (with encryption in picture) on whether
+        # the model downloaded/present with previous framework execution would be plain or encrypted model.
+        # So, we download fresh model everytime we execute the test.
+        try:
+            os.remove("data/" + TF_BERT_FP2_MDOEL_NAME)
+        except OSError:
+            pass
+        print("\n-- Downloading BERT FP32 model..")
+        utils.exec_shell_cmd(TF_BERT_FP32_MODEL_WGET_CMD, None)
+        
+        print("\n-- Required BERT models downloaded..")
 
-    def download_resnet_models(self):
+    def download_resnet_models(self, test_config_dict):
         if not os.path.exists('./models'):
             print("\n-- Downloading RESNET Intel_AI models..")
             utils.exec_shell_cmd(TF_RESNET_INTEL_AI_MODELS_CLONE_CMD, None)
 
-        resnet_inte8_model_name = os.path.basename(TF_RESNET_INT8_MODEL_WGET_CMD.split()[1])
-        if not os.path.exists(resnet_inte8_model_name):
-            print("\n-- Downloading RESNET Pretrained model..")
-            utils.exec_shell_cmd(TF_RESNET_INT8_MODEL_WGET_CMD, None)
+        # We are deleting the model to avoid complication (with encryption in picture) on whether
```

Comment:
We may not need to delete the base model, as its the same version.

Reply:
Done. Earlier, was thinking to use the same path for encryption as well. But, then after changing the paths for plain and encrypted models, forgot to remove this snippet.